node_modules/whatsapp-web.js/index.d.ts:1447:6 - error TS1036: Statements are not allowed in ambient contexts.

### Is there an existing issue for this?

- [X] I have searched the existing issues

### Describe the bug

Trying to build whatsapp web as dependency in a project TS/JS for running in docker throws

node_modules/whatsapp-web.js/index.d.ts:1447:6 - error TS1036: Statements are not allowed in ambient contexts.

The latest stable version throws several ones, and It was solved in main branch. But It stills there is a one more issue.

nest build
node_modules/whatsapp-web.js/index.d.ts:1447:6 - error TS1036: Statements are not allowed in ambient contexts.

1447     };
          ~

Found 1 error(s).

### Expected behavior

building successfuly

### Steps to Reproduce the Bug or Issue

nest build

### Relevant Code

_No response_

### Browser Type

Chromium

### WhatsApp Account Type

Standard

### Does your WhatsApp account have multidevice enabled?

Yes, I am using Multi Device

### Environment

node 18 slim
mac m2
branch main


### Additional context

_No response_

--- a/index.d.ts
+++ b/index.d.ts
@@ -1422,7 +1422,7 @@ declare namespace WAWebJS {
             message: string;
             isInviteV4Sent: boolean,
         }
-    };
+    }
 
     /** An object that handles options for adding participants */
     export interface AddParticipantsOptions {
@@ -1446,7 +1446,7 @@ declare namespace WAWebJS {
          * @default ''
          */
         comment?: string
-    };
+    }
 
     /** An object that handles the information about the group membership request */
     export interface GroupMembershipRequest {

